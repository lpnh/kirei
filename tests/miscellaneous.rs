mod helper;

#[test]
fn empty_file() {
    format_and_snapshot!("empty_file.html");
}
