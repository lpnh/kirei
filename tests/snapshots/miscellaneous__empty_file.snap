---
source: tests/miscellaneous.rs
description: ""
---

